php-practice/section10.php

```diff
@@ -76,6 +89,16 @@
  * associative array with `name` and `genre` indexes) and loops over it, printing out the name of each other.
  */
 
+function authorName(array $favoriteAuthor) {
+      foreach ($favoriteAuthor as $x => $x_value) {
+      echo $x_value['name'];
+      echo PHP_EOL;
+    }
+}
+
+authorName($favoriteAuthor);
+/**I got this one to work; however, I don't understand why $x_value['name'] worked. I had assumed that $x[$x_value] would work and tried $x_value['name'] on a whim.
+Could you explain what the difference is?
```

Comment:
With this syntax:

```php
foreach ($fooArray as $fooKey => $fooValue) {
  // ...
}
```

Remember, `$fooArray` is the array you want to loop over, `$fooKey` is a key (or **index**) defined in `$fooArray`, and `$fooValue` is the value assigned to that index.